src/store/modules/current-company/__tests__/reducer.test.js

```diff
@@ -0,0 +1,17 @@
+import currentCompanyReducer, {INITIAL_STATE} from 'store/modules/current-company/reducer';
+import {updateCompany} from 'store/modules/current-company/actions'
+
+describe('Current company reducer', () => {
+  it('returns unmodified state', () => {
+    expect(currentCompanyReducer(undefined, {})).toStrictEqual({ id: 0, name: '' })
+  })
```

Comment:
Following the same idea as https://github.com/comarev/comarev-dashboard/pull/95#discussion_r906032392, I think there is some room for improvements in this test.

Something to be improved:
1. I noticed we pass `undefined` as 1st param and `{}` as 2nd param, but why? What do they mean? We could name these arguments so that it makes the test easy to read;
2. The test message says `returns unmodified state` but why? Maybe it is worth saying the reason that this is happening, something like `returns unmodified state when the action is not valid`, that way everyone would understand why that happens. 

So I'd suggest doing something like:
```suggestion
  it('returns unmodified state when the action is not valid', () => {
    const state = { id: 10, name: 'My Company' };
    const action = {};

    const expectedState = { id: 10, name: 'My Company' };
    const returnedState = currentCompanyReducer(state, action);

    expect(returnedState).toStrictEqual(expectedState);
  });
```